src/results/electric_storage.jl

```diff
@@ -49,10 +54,10 @@ function add_electric_storage_results(m::JuMP.AbstractModel, p::REoptInputs, d::
 
     if r["size_kwh"] != 0
     	soc = (m[Symbol("dvStoredEnergy"*_n)][b, ts] for ts in p.time_steps)
-        r["year_one_soc_series_fraction"] = round.(value.(soc) ./ r["size_kwh"], digits=3)
+        r["soc_series_fraction"] = round.(value.(soc) ./ r["size_kwh"], digits=3)
 
         discharge = (m[Symbol("dvDischargeFromStorage"*_n)][b, ts] for ts in p.time_steps)
-        r["year_one_to_load_series_kw"] = round.(value.(discharge), digits=3)
+        r["storage_to_load_series_kw"] = round.(value.(discharge), digits=3)
```

Comment:
I'm generally fine with this "storage_to..." which is currently consistent between electric, hot, and cold storage, where the "electric" vs "thermal" is not specified (not that you asked :-)).

Reply:
Ok! I guess the alternative would be "electric_storage_to" and "thermal_storage_to". But if you're good with just "storage" we can move forward with that.